### ISSUE TITLE
[BUG][Android]: In HLS playlist videos , the video suddenly gets bigger

### ISSUE THREAD

**Issue by bekircanbozkurt** [TheWidlarzGroup/react-native-video#3620]
### Version

v6 (Beta)

### What platforms are you having the problem on?

Android

### Architecture

Old architecture

### What happened?

When I try to play HLS playlist videos, the video suddenly becomes larger after a few seconds. And after this happens the video starts to stutter

### Reproduction

repository link

### Code Sample

```
const VideoPlayer = () => {
    const videoSrc = "https://bitdash-a.akamaihd.net/content/sintel/hls/playlist.m3u8";
    return (
        <Video
        source={{ uri:videoSrc }}
        style={styles.backgroundVideo}
        />
    )
}
```
### Screenrecord

https://github.com/react-native-video/react-native-video/assets/58627732/5435e394-a3cd-490f-adbc-fb39fe614726

_**The video grows around the 14th second.**_

**Comment by freeboub** [comment]
@bekircanbozkurt We have this uri in sample app, but we don't have this issue...
can you provide style backgroundVideo please ?
Do you reproduce it in the sample ?

**Comment by bekircanbozkurt** [comment]
> @bekircanbozkurt We have this uri in sample app, but we don't have this issue... can you provide style backgroundVideo please ? Do you reproduce it in the sample ?

Of course @freeboub . I have the same problem in the sample app.

`
backgroundVideo: {
        flex:1,
    },
`

**Comment by freeboub** [comment]
@bekircanbozkurt Do you have it on a real device or only in emulator ? It is possibly an emulator issue...
Notice a ne patch ongoing can fix the issue (but not sure).

**Comment by seyedmostafahasani** [comment]
@bekircanbozkurt 
Can you use width and height instead of flex? For example: style={{width: SCREEN_WIDTH, height: SCREEN_HEIGHT}}

**Comment by dev-folks** [comment]
@bekircanbozkurt
Hello! Have you found the cause and solution to this problem?

**Comment by freeboub** [comment]
It is possible that next release will fix this issue (6.7.0 I think).
Let me know if it is reproduced.
Notice also that I found a similar bug on emulator
